app/src/test/java/tech/relaycorp/courier/domain/StoreMessageTest.kt

```diff
@@ -60,57 +54,63 @@ internal class StoreMessageTest {
         val noStorageSpace = StorageUsage(StorageSize.ZERO, StorageSize.ZERO)
         whenever(getStorageUsage.get()).thenReturn(noStorageSpace)
 
-        assertNull(subject.storeCargo(buildMessageStream()))
+        assertNull(subject.storeCargo(RAMFMessageFactory.buildCargoSerialized().inputStream()))
         verify(diskRepository, never()).writeMessage(any())
         verify(storedMessageDao, never()).insert(any())
     }
 
     @Test
-    internal fun `store cargo with malformed message`() = runBlockingTest {
-        whenever(cargoDeserializer.invoke(any())).then {
-            throw RAMFMessageMalformedException()
-        }
-
-        assertNull(subject.storeCargo(buildMessageStream()))
+    internal fun `do not store malformed cargo`() = runBlockingTest {
```

Comment:
You'll notice I replaced "store" with "do not store" a few times. I think both are valid, it just depends on whether you want to describe the scenario or the expected outcome. I simply renamed them because some tests above did start with "do not".